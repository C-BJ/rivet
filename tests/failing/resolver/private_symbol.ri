mod my_mod {
    struct PrivateStruct {
        a: i32;
        b: PrivateEnum;
    }
    const PRIVATE_CONST: i32 = 1;

    enum PrivateEnum { Variant }
}

fn main() {
    _ = my_mod::PrivateStruct{
        a: my_mod::PRIVATE_CONST,
        b: my_mod::PrivateEnum::Variant
    };
}
