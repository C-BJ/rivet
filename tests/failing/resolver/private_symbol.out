tests/failing/resolver/private_symbol.ri:12:17: error: type `PrivateStruct` is private
  12 |     _ = my_mod::PrivateStruct{
     |                 ^
tests/failing/resolver/private_symbol.ri:13:20: error: constant `PRIVATE_CONST` is private
  13 |         a: my_mod::PRIVATE_CONST,
     |                    ^
tests/failing/resolver/private_symbol.ri:14:20: error: type `PrivateEnum` is private
  14 |         b: my_mod::PrivateEnum::Variant
     |                    ^
rivetc: error: could not compile package `private_symbol`, aborting due to 3 previous errors.